common/log.cpp

```diff
@@ -76,4 +79,7 @@ void yamiTraceInit()
         }
         isInit = 1;
     }
+#ifdef __ENABLE_DEBUG__
+     fprintf(stderr, "yami log isn't enabled (--enable-debug)\n");
```

Comment:
I think it'd be better to only print this if the yamiLogFlag > YAMI_LOG_ERROR since YAMI_LOG_ERROR always gets logged regardless of debug or not.